src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,15 @@
 // Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
+import './TodoInfo.scss';
+
+export const TodoInfo = ({ todo }) => (
+  <article className={cn('TodoInfo', {
+    'TodoInfo--completed': todo.completed,
+  })}
+  >
+    <h2 className="TodoInfo__title">{todo.title}</h2>
+
+    <UserInfo user={todo.user} />
```

Comment:
Can be case when your user will be `null` (check your `getUserById` function).

So you need to add check on `user` before render `<UserInfo />`:
```suggestion
    {todo.user && <UserInfo user={todo.user} />}
```